scripts/deploy-kubevirt-and-cdi.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+function latest_version() {
+  local repo="$1"
+  curl -s https://api.github.com/repos/kubevirt/${repo}/releases/latest | \
+    grep '"tag_name":' | \
+    sed -E 's/.*"([^"]+)".*/\1/'
+}
+
+
+# Deploying kubevirt
+KUBEVIRT_VERSION=$(latest_version "kubevirt")
+KUBEVIRT_NAMESPACE=${1:-kubevirt}
+
+kubectl apply -n ${KUBEVIRT_NAMESPACE} -f "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml"
+kubectl apply -n ${KUBEVIRT_NAMESPACE} -f "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml"
+
+kubectl patch -n ${KUBEVIRT_NAMESPACE} kubevirt kubevirt --type='json' -p '[{
+  "op": "add",
+  "path": "/spec/configuration/developerConfiguration/featureGates/-",
+  "value": "NUMA",
+},{
+  "op": "add",
+  "path": "/spec/configuration/developerConfiguration/featureGates/-",
+  "value": "GPU",
+}]'
+
+echo "Waiting for Kubevirt to be ready..."
+kubectl wait --for=condition=Available --timeout=600s -n ${KUBEVIRT_NAMESPACE} kv/kubevirt
+
+
+# Deploying CDI
+CDI_VERSION=$(latest_version "containerized-data-importer")
```

Comment:
I don't think we have a version of `stable.txt` available for CDI so this is fine.